README.md

```diff
@@ -239,30 +317,84 @@ can determine the _order_ in which they need to be worked on.
 
 ### C) _Estimate_ the Time Required to Perform the Work
```

Comment:
Shouldn't it be the person who is going to carry out the work who estimates it? The implication here is that the person who confirms the need and assigns a priority is doing the estimating.